applications/HDF5Application/tests/test_xdmf_output.py

```diff
@@ -0,0 +1,161 @@
+        self.currentPath = os.getcwd()
+        self.scope = scope
+
+    def __enter__(self):
+        os.chdir(self.scope)
+
+    def __exit__(self, type, value, traceback):
+        os.chdir(self.currentPath)
+
+class TestCase(KratosUnittest.TestCase):
+
+    def setUp(self):
+        pass
+
+    def _remove_file(self, file_path):
+        if os.path.isfile(file_path):
+            os.remove(file_path)
+    
+    def _initialize_model_part(self, model_part):
+        num_tri_elems = 10
+        num_quad_elems = 15
+        num_tri_conds = 5
+        num_quad_conds = 10
+        num_nodes = (num_tri_elems + 2) + (num_quad_elems + 2)
+        # Add variables.
+        model_part.AddNodalSolutionStepVariable(DISPLACEMENT)
+        model_part.AddNodalSolutionStepVariable(DENSITY)
+        model_part.AddNodalSolutionStepVariable(PARTITION_INDEX)
+        # Create nodes.
+        for i in range(num_nodes):
+            x = 0.0
+            y = 0.0
+            z = 0.0
+            model_part.CreateNewNode(i + 1, x, y, z)
+        # Create triangle elements.
+        for i in range(num_tri_elems):
+            elem_id = i + 1
+            node_ids = [i + 1, i + 2, i + 3]
+            prop_id = 1
+            prop = model_part.GetProperties()[prop_id]
+            model_part.CreateNewElement("Element2D3N", elem_id, node_ids, prop)
+        # Create quad elements.
+        for i in range(num_quad_elems):
+            elem_id = num_tri_elems + i + 1
+            node_ids = [num_tri_elems + i + 1, num_tri_elems + i + 2, num_tri_elems + i + 3, num_tri_elems + i + 4]
+            prop_id = 1
+            prop = model_part.GetProperties()[prop_id]
+            model_part.CreateNewElement("Element2D4N", elem_id, node_ids, prop)
+        # Create triangle conditions.
+        for i in range(num_tri_conds):
+            cond_id = i + 1
+            node_ids = [i + 1, i + 2, i + 3]
+            prop_id = 1
+            prop = model_part.GetProperties()[prop_id]
+            model_part.CreateNewCondition("SurfaceCondition3D3N", cond_id, node_ids, prop)
+        # Create quad conditions.
+        for i in range(num_quad_conds):
+            cond_id = num_tri_conds + i + 1
+            node_ids = [num_tri_conds + i + 1, num_tri_conds + i + 2, num_tri_conds + i + 3, num_tri_conds + i + 4]
+            prop_id = 1
+            prop = model_part.GetProperties()[prop_id]
+            model_part.CreateNewCondition("SurfaceCondition3D4N", cond_id, node_ids, prop)
+        model_part.SetBufferSize(2)
+
+    def _get_file(self):
+        params = Parameters("""
+        {
+            "file_name" : "test_xdmf_output.h5",
+            "file_access_mode" : "truncate",
+            "file_driver" : "sec2"
+        }""")
+        return HDF5FileSerial(params)
+
+    def _get_model_part_io(self, hdf5_file):
+        params = Parameters("""
+        {
+            "prefix" : "/ModelData",
+            "list_of_elements" : ["Element2D3N", "Element2D4N"],
```

Comment:
I believe this is not needed any more right?

Reply:
correct